### PR TITLE
Update the species number counts

### PR DESCRIPTION
**Pull request by jshoe**
- Adds feature to update the counts in searches.
- Not sure if there's a better way to reset the counts?

mar-20-2018 16-24-08

### CODE REVIEW

File: app/assets/src/components/PipelineSampleReport.jsx [hunk 1]
```diff
@@ -253,6 +253,8 @@ class PipelineSampleReport extends React.Component {
         const taxon = this.state.taxonomy_details[i];
         if (taxon.genus_taxid == taxon.tax_id) {
           if (matched_taxons.length > 0) {
+            // Update species count on page after a search
```
Comment (yunfangjuan) [inline app/assets/src/components/PipelineSampleReport.jsx]: line 270 needs the same condition.

File: app/assets/src/components/PipelineSampleReport.jsx [hunk 2]
```diff
@@ -297,15 +299,27 @@ class PipelineSampleReport extends React.Component {
         }
       }
     } else {
-      selected_taxons = thresholded_taxons;
+      // Called for All search
+      let included = thresholded_taxons;
+      for (let i = 0; i < included.length; i++) {
```
Comment (yunfangjuan) [inline app/assets/src/components/PipelineSampleReport.jsx]: hm this doesn't apply to excludeCategory filters?

**Comment by jshoe** [comment]
Ah OK I was a little unclear on the exact logic

**Comment by jshoe** [comment]
@yunfangjuan Updated. Makes much more sense now

**Comment by jshoe** [comment]
@cdebourcy

**Pull request merged**